tensorizer/stream_io.py

```diff
@@ -137,6 +138,7 @@ def __init__(
             "Accept-Encoding: identity",
             "-s",
             "-f",
+            "--raw",
```

Comment:
This flag will lead to incorrect behaviour with a chunked `Transfer-Encoding`.
`Accept-Encoding: identity` only affects `Content-Encoding`, but `--raw` affects interpretation of both.
```suggestion
```